packages/fxa-auth-server/test/remote/recovery_code_tests.js

```diff
@@ -11,248 +11,254 @@ const Client = require('../client')();
 const otplib = require('otplib');
 const BASE_36 = require('../../lib/routes/validators').BASE_36;
 
-[{version:""},{version:"V2"}].forEach((testOptions) => {
+[{ version: '' }, { version: 'V2' }].forEach((testOptions) => {
```

Comment:
Is it possible to undo the formatting, so we can see what changed? ie remove the indent?